src/Data/Empty/Polymorphic.agda

```diff
@@ -8,9 +8,12 @@
 
 module Data.Empty.Polymorphic where
 
+import Data.Empty as ⊥
 open import Level
 
-data ⊥ {ℓ : Level} : Set ℓ where
+⊥ : {ℓ : Level} → Set ℓ
+⊥ {ℓ} = Lift ℓ ⊥.⊥
 
-⊥-elim : ∀ {w ℓ} {Whatever : Set w} → ⊥ {ℓ} → Whatever
-⊥-elim ()
+-- make ⊥-elim-P dependent too, as it does seem useful
+⊥-elim-P : ∀ {w ℓ} {Whatever : ⊥ {ℓ} → Set w} → (witness : ⊥ {ℓ}) → Whatever witness
```

Comment:
Hmm do we need the `-P`. We shouldn't really need to import both the polymorphic and the non-polymorphic variety at the same time right?

Reply:
I'll remove the `-P` and see. There was one file with a weird problem, but I'm sure I can fix it.